openmdao/docs/openmdao_book/features/building_blocks/components/metamodelsemistructured_comp.ipynb

```diff
@@ -0,0 +1,492 @@
+    "\n",
+    "`MetaModelSemiStructuredComp` handles semi-structured data by treating every \"column\" of data beyond the first dimension independently.  In the figure above, it doesn't matter that two adjacent columns have different points, as the interpolation in 'y' happens independently in each column before those values are subsequently interpolated in 'x'. \n",
+    "\n",
+    "The following interpolation methods are supported in `MetaModelSemiStructuredComp`:\n",
+    "\n",
+    "```{eval-rst}\n",
+    "+---------------+--------+------------------------------------------------------------------+\n",
+    "| Method        | Order  | Description                                                      |\n",
+    "+===============+========+==================================================================+\n",
+    "| slinear       | 1      | Basic linear interpolation                                       |\n",
+    "+---------------+--------+------------------------------------------------------------------+\n",
+    "| lagrange2     | 2      | Second order Lagrange polynomial                                 |\n",
+    "+---------------+--------+------------------------------------------------------------------+\n",
+    "| lagrange3     | 3      | Third order Lagrange polynomial                                  |\n",
+    "+---------------+--------+------------------------------------------------------------------+\n",
+    "| akima         | 3      | Interpolation using Akima splines                                |\n",
+    "+---------------+--------+------------------------------------------------------------------+\n",
+    "```\n",
+    "\n",
+    "Note that `MetaModelSemiStructuredComp` only accepts scalar inputs and outputs. If you have a multivariable function, each input variable needs its own named OpenMDAO input.\n",
+    "\n",
+    "For multi-dimensional data, fits are computed on a separable per-axis basis. A single interpolation method is used for all dimensions, so the minimum table dimension must be high enough to use the chosen interpolate. Further, the minimum dimension will potentially be driven by the sparsest area of your data, so care should be taken to choose the smallest regular dimension as the first dimension.\n",
+    "\n",
+    "Extrapolation is supported, and is enabled by default. Given the move sparse nature of semi-structured data, we expect that extrapolation will occur more often. When evaluating a point, if any of the requested values falls beyond the first or last point in a row, you are extrapolating. There is always a risk when extrapolating, so it is important to be careful. Extrapolation can be disabled via the `extrapolate` option (see below).\n",
```

Comment:
typo: `move sparse`

Reply:
Fixed both typos and pushed up for verification.  Note: holding this until I finish my improvements.